x-pack/plugins/security_solution/public/common/components/exceptions/use_add_exception.tsx

```diff
@@ -111,6 +116,23 @@ export const useAddOrUpdateException = ({
           });
         }
 
+        if (shouldBulkClose === true) {
+          const filter = getQueryFilter(
+            '',
+            'kuery',
+            [],
+            ['.siem-signals'],
+            exceptionItemsToAddOrUpdate,
+            false
+          );
+          await updateAlertStatus({
```

Comment:
@FrankHassanabad I think we may need to use `wait_for_completion=false` on the `updateByQuery` [here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/open_close_signals_route.ts#L56). Otherwise the request could take a while.